commands/fcplink.js

```diff
@@ -0,0 +1,18 @@
+const { SlashCommandBuilder } = require('discord.js')
+const { ActionRowBuilder, ButtonBuilder, ButtonStyle } = require('discord.js')
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('fcplink')

```

Comment:
Change this to something like "flight-crew-portal" for easy member use. Someone is gonna say it and thats what most people are going to type initally (becuase thats what I did)